site/content/blog/2025-07-21-timing-rule-changes/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Timing Related Scan Rule Changes"
+summary: >
+  Scan rules related to timing attacks have been split ore renamed.
+images:
+- https://www.zaproxy.org/blog/2025-07-21-timing-rule-changes/something.png
+type: post
+tags:
+- blog
+- browser
+date: "2025-07-21"
+authors:
+- thorin
+---
+
+### Time Based Scan Rule Changes
+
+We have recently implemented a number of [requested](https://github.com/zaproxy/zap-extensions/pull/4316) changes with regard to scan rules which perform time based attacks.
+
+![ZAPbot with Edge browser icon](images/blog-edge.png)
+
+These changes were made to provide users with greater flexibility in creating scan policies.
+In the last few years a number of changes have been implemented to address the potential for false positives.&sup1;
+
+Here is a breakdown of what was changed:
+
+* The following Rules and Alerts were renamed to indicate that they're all Time Based (their IDs have stayed the same):
+    * [40019](/docs/alerts/40019/) - SQL Injection - MySQL
+    * [40020](/docs/alerts/40020/) - SQL Injection - Hypersonic
+    * [40021](/docs/alerts/40021/) - SQL Injection - Oracle*
+    * [40022](/docs/alerts/40022/) - SQL Injection - PostgreSQL
+    * [40024](/docs/alerts/40024/) - SQL Injection - SQLite* (This rule had contained code for union based SQLi attacks however it had been disabled for a long time. If further SQLite attacks are added in the future they will appear associated with rule ID 90038.)
+    * [40027](/docs/alerts/40027/) - SQL Injection - MsSQL
+* [40033](/docs/alerts/40033/) - NoSQL Injection  MongoDB - Timing attacks were previously split into rule [90039](/docs/alerts/90039/).
+* [90020](/docs/alerts/90020/) - Remote OS Command Injection - Time based attacks and Alerts were split into a separate rule with ID [90037](/docs/alerts/90037/).
+
+#### Alert Tag TEST_TIMING
+
+The rules (and alerts) are now also decorated with the [TEST_TIMING](/alerttags/test_timing/)[Alert Tag](/docs/desktop/ui/dialogs/addalert/#alert-tags).
+
+### New Passive Scan Rule - ZAP is Out of Date
```

Comment:
I'm now having second thoughts about including this - it doesnt seem to be releated to the rest of the post?